shacl12-profiling/index.html

```diff
@@ -436,32 +436,103 @@
+				<dt><a href="https://w3c.github.io/data-shapes/shacl12-node-expr/">SHACL 1.2 Node Expressions</a></dt>
+				<dd>defines graph expressions used to determine focus nodes in SHACL</dd>
+				<dt><a href="https://w3c.github.io/data-shapes/shacl12-inf-rules/">SHACL 1.2 Inference Rules</a></dt>
+				<dd>defines SHACL's methods of rule-based inference</dd>
+				<dt><a href="https://w3c.github.io/data-shapes/shacl12-ui/">SHACL 1.2 UI</a></dt>
+				<dd>defines SHACL's use for User Interface generation</dd>
+				<dt><a href="https://w3c.github.io/data-shapes/shacl12-compact-syntax/">SHACL 1.2 Compact Syntax</a></dt>
+				<dd>defines an RDF syntax for expressing SHACL concepts</dd>
+				<dt>SHACL 1.2 Profiling (this specification)</dt>
+				<dd>defines the use of SHACL for profiling data, including SHACL data</dd>
+			</dl>
+		</section>
 
-			<section id="terminology">
-				<h3>Terminology</h3>
+		<section class="introductory">
+			<h2>Document Outline</h2>
+			<p>
+				The introduction provides background concepts of profiling and states this specification's scope.
+			</p>
+			<p>
+				Sections 2 & 3 cover the two main elements within the stated scope.
+			</p>
+		</section>
 
+		<section id="introduction">
+			<h2>Introduction</h2>
+			<section id="what-is">
+				<h3>What is profiling?</h3>
 				<p>
-					Terminology used throughout this document is consistent with several sources:
+					Profiling is the act of creating a "profile" of something.
+				</p>
+				<p>
+					Generically, in English, a "profile" of something is as follows:
+				</p>
+				<p style="padding: 0 50px 0 50px;">
+					The outline of a physical object or feature, or a representation of this
+					<br /><br />- Oxford English dictionary, use of the word "profile" since the 17th century
+				</p>
+				<p>
+					Within the world of data, a derived definition of "profile" consistent with the above is —
+				</p>
+				<p>
+					A summary or an extraction
+				</p>
+				<p>
+					In this definition, the essence of the English word is retained, since a summary or extraction of or from a data object <em>may</em> be an outline of it; for example, a 2D representation of a 3D spatial object. or a statistical summary of a dataset having lots of parts.
+				</p>
+				<p>
+					By definition, SHACL constrains (RDF) data; thus, any data that is valid according to a shapes graph will be a profile of the data graph that was validated. In the case of a shapes graph validating all elements of a data graph, the valid data will be a "null" profile of the data graph, that is identical to it.
+				</p>
+				<p>
+					The W3C's <em>Profiles Vocabulary</em> [[dx-prof]] has defined "data profiling" in the context of <em>specifications</em> or <em>data specifications</em>:
+				</p>
+				<p style="padding: 0 50px 0 50px;">
+					<em>A data specification that constrains, extends, combines, or provides guidance or explanation about the use of other data specifications.</em>
```

Comment:
```suggestion
					<em>A data specification that constrains, extends, combines, or provides guidance or explanation about the use of other data specifications. This definition includes what are sometimes called "application profiles", "metadata application profiles", or "metadata profiles".</em>
```
Should we add the other variants referred to in the specification?

Reply:
I'd prefer not to as SHACL will not know the difference between a "profile" and a "metadata profile": it will just generically profile whatever people want it to.